lang/c++/impl/json/JsonIO.hh

```diff
@@ -415,9 +431,10 @@ public:
     void encodeNumber(double t) {
```

Comment:
We can have a single specialization for `float` and `double`:

```
template<typename T>
encodeNumber(typename std::enable_if<std::is_floating_point<T>::value, T>::type t) {
    // implementation common to float/double
}

template<typename T>
encodeNumber(typename std::enable_if<!std::is_floating_point<T>::value, T>::type t) {
    // implementation for all other types
}

```

Reply:
Done. Please take a look again. Thanks!